keanu-python/build.gradle

```diff
@@ -6,7 +6,7 @@ import static org.apache.tools.ant.taskdefs.condition.Os.isFamily
 task installPipenv {
     doLast {
         exec {
-            commandLine 'pip', 'install', 'pipenv==2018.10.13'
+            commandLine 'pip', 'install', 'pipenv==2018.11.26'
```

Comment:
We could drop the version number (i.e. always install latest). 

The version number was needed temporarily when we first set up our python build, because coincidentally at that time a new release of pip created a bug that caused pipenv to fail.

Reply:
Agree that would probably be fine. Given the flakiness that I've seen between `pipenv` version, I think I'd be in favour of keeping the explicit version check for now. Maybe things have got better since but see e.g. https://chriswarrick.com/blog/2018/07/17/pipenv-promises-a-lot-delivers-very-little/#the-break-neck-pace-of-pipenv

Reply:
fine by me